radiography/build.gradle.kts

```diff
@@ -22,9 +22,7 @@ plugins {
 }
 
 android {
-  // Using 28 for now to get the Robolectric tests working.
   compileSdkVersion(30)
-  buildToolsVersion = "29.0.2"
```

Comment:
This has been optional since AGP 3, and so instead of upgrading this to the latest version I just removed it.